packages/env/lib/config.js

```diff
@@ -400,7 +400,9 @@ function getHomeDirectory() {
 	 */
 	return path.resolve(
 		os.homedir(),
-		os.platform() === 'linux' ? 'wp-env' : '.wp-env'
+		!! ( await fs.stat( '/snap' ).catch( () => false ) )
```

Comment:
Full disclosure: I'm not in love with this either, but apparently `fs.promises.exists` does not exist (no pun intended) so this is really the only way to do this in a promise-based way.

Reply:
is `fs` imported in this file yet? the unit tests are complaining about `TypeError: fs.stat is not a function`

Reply:
Yes it is.

https://github.com/WordPress/gutenberg/blob/3081e82f6c4d31166cbe2a642ece417d644653b5/packages/env/lib/config.js#L5

`fs.promises.stat` is indeed a function. So perhaps a node version issue?

Reply:
Pretty sure the updated tests fixes this. Unresolve if not.